repoparse/internal/classes.py

```diff
@@ -0,0 +1,230 @@
+from dataclasses import dataclass
+from os import path
+import sys
+from typing import Dict, List, Union, cast
+
+from dulwich.patch import gen_diff_header, is_binary, patch_filename, \
```

Comment:
wrong order of imports. It should be:
* standard library
* external dependencies
* from this module